packages/plugin-ext/src/plugin/tree/tree-views.ts

```diff
@@ -88,6 +89,22 @@ export class TreeViewsExtImpl implements TreeViewsExt {
             get onDidChangeVisibility() {
                 return treeView.onDidChangeVisibility;
             },
+            // tslint:disable-next-line:typedef
```

Comment:
or just type string everywhere? it should be even shorter

Reply:
I have removed typedefs in favour of specifying type string for getters